docs/source/administrator/management/users.md

```diff
@@ -10,22 +10,31 @@ FlowAuth is the tool which analysts will use to generate tokens which will allow
 
 2. Under "Servers", add a new server by clicking the '+' button, uploading the spec downloaded from the server, and setting the latest expiry and longest life for tokens.
 
-3. Enable or disable permissions for this server under "Available API scopes".
+5. Create a new role for the server
```

Comment:
The formatting on this section is wonky in the built docs:

<img width="738" alt="Screenshot 2023-02-17 at 09 56 17" src="https://user-images.githubusercontent.com/472828/219612875-38f89b3e-f3af-48ce-9f9f-ac90f937912b.png">